edina/src/data/Su/json/exhibit.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "title": "“漆语”院藏当代漆画展",
+        "href": "https://m-exhibit.artron.net/exhibition-35858.html",
+        "img": "20150710051708339_副本.jpg",
+        "location": "广州艺术博物院",
+        "time": "2015-07-15至2015-09-13",
+        "detail": "由广州艺术博物院主办的“漆语——院藏当代漆画展”2015年7月15日...详细"

```

Comment:
Missing full detail

Reply:
no full detail, I will search and  fill it out manually

Reply:
Let's create a new issue on this AI